mappings/net/minecraft/client/util/ParticleUtil.mapping

```diff
@@ -4,12 +4,12 @@ CLASS net/minecraft/class_5945 net/minecraft/client/util/ParticleUtil
 		ARG 1 pos
 		ARG 2 direction
 		ARG 3 effect
-	METHOD method_34682 spawnParticle (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2394;Lnet/minecraft/class_4801;)V
+	METHOD method_34682 spawnParticle (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2394;Lnet/minecraft/class_6019;)V
 		ARG 0 world
 		ARG 1 pos
 		ARG 2 effect
 		ARG 3 range
-	METHOD method_34683 spawnParticle (Lnet/minecraft/class_2350$class_2351;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;DLnet/minecraft/class_2394;Lnet/minecraft/class_4801;)V
+	METHOD method_34683 spawnParticle (Lnet/minecraft/class_2350$class_2351;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;DLnet/minecraft/class_2394;Lnet/minecraft/class_6019;)V
```

Comment:
is this right?

Reply:
Yes the methods changed in 21w13a but the PR was created in 21w11a